src/azure-cli/azure/cli/command_modules/vm/__init__.py

```diff
@@ -1,7 +1,14 @@
-# --------------------------------------------------------------------------------------------
+# --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
+# Licensed under the MIT License. See License.txt in the project root for
```

Comment:
is this auto-generated? what's the general rule for updating __init__.py?

Reply:
Yes. It looks good. Just keep it.